### DISCUSSION TITLE
Easier Build Instructions on Windows.

### DISCUSSION THREAD

**Discussion by JaminShanti** [jamulussoftware/jamulus#2745] [Documentation]
A while ago I wanted to help with the CD of Jamulus.  I never was able to build it on windows.  Gave up trying.  Can anyone improve the Build instructions on windows?  I felt like I was in the Unicorn Project.

**Comment by pljones** [comment]
Install Qt Creator from the Qt website, along with the relevant dependencies for the build you want.

Open the project in Qt Creator and configure the build you want.

Hit Build.

Wait.

Done.

----

Alternative:

Fork the project on Github.

Use the Auto-Build on Github to build your Windows versions (and all the others).

**Reply by JaminShanti** [reply]
Press a button
Neat.
Can someone else reply?

**Reply by ann0see** [reply]
I got it to work (via deploy_windows.ps1). Probably @pgScorpio knows more about it. Where are you stuck?

**Reply by JaminShanti** [reply]
That's what I used back last year. Maybe it has some new commits?

**Reply by pgScorpio** [reply]
If you are used to visual studio you can also run:
`qmake -tp vc Jamulus.pro`
This will create the vcxproj files which enables you to build and debug in visual studio.

**Reply by ann0see** [reply]
Yes, the ASIO files moved to libs/ and some smaller other changes. Can you describe exactly what goes wrong

**Reply by JaminShanti** [reply]
I haven't done this in a year, is there  a page that tell dependencies? for Visual Studio version and QT?  I remember last time the QT version that worked was behind a pay wall.

**Reply by ann0see** [reply]
Just use the open source version of Qt5 or Qt6. Dependencies should be the default ones. 

See the windows.ps1 https://github.com/jamulussoftware/jamulus/tree/master/.github/autobuild for more information how the CI does it. 

Edit: could you also please say exactly what needs to be improved about the compilation instructions for windows here such that we can edit them: https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md#windows
Since on Windows the CLI is not that often used, giving exact commands you should type into the terminal similar to macOS/Linux documentation is not that detailed and hands on unfortunately.

**Reply by henkdegroot** [reply]
Pleas note that you manually need to download the ASIO SDK (if you don't use the deploy_windows.ps1 script, this script will automatically download it).
